src/Main.java

```diff
@@ -1,7 +1,27 @@
 public class Main {
 
     public static void main(String[] args) {
+        String str = "Was it a cat I saw";
+        checkPalindrome(str);
+    }
+
 
-        System.out.println("Hello World!");
+    public static void checkPalindrome(String str) {
+        String allLowStr = str.toLowerCase();
+        String cleanStr = allLowStr.replaceAll("\\s+", "");
```

Comment:
Everything is fine, but you did not handle special characters(like: "$#&^%*()")

Reply:
Hi, Shall I use this : string.replaceAll("[^\\W\\S]","")?

Reply:
Hi, I think I know. I can use''\\W" to replace all the nonword character.